safe/report/impact_report.py

```diff
@@ -440,6 +443,14 @@ def impact_function(self):
         """
         return self._impact_function
 
+    @property
+    def multi_exposure_impact_function(self):
```

Comment:
Is it needed? Is this empty when it's not a multi_exposure dialog?

Reply:
yes it's needed, and it's empty when it's not a multi_exposure dialog

Reply:
Ok, not a fan of specific properties which belong to the parent caller. But if you say it's needed ...

Reply:
so what I did here is to avoid the calling of `ImpactReport.impact_function` (it was being used a lot in the extractors), because if we put the `MultiExposureImpactFunction()` as a value of `impact_function` parameter in the `ImpactReport` it will give error because `MultiExposureImpactFunction()` doesn't have a complete attribute as an 'ImpactFunction()', unless `MultiExposureImpactFunction()` is inherited from `ImpactFunction()`. any thoughts?

Reply:
No, let's go for it. It's not inheriting. The multiexposure is on top of the impact function now

Reply:
thanks @Gustry I also wish for a better solution for this. Maybe I will just put a note why we do it for now